README.md

```diff
@@ -55,11 +55,13 @@ instance-1  us-central1-a  n1-standard-1               10.240.0.2   8.34.219.29
 
```

Comment:
ok, added some text to build the image supporting appengine-java and kubectl in one dockerfile